File: src/main/java/org/shark/example/datasource/cassandra/pojo/MessageDo.java
package org.shark.example.datasource.cassandra.pojo;

import lombok.Data;
import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

import java.util.Date;

@Data
@Table("message")
public class MessageDo {

    @PrimaryKey
    private MessageKeyDo messageKey;

    @Column("time")
    private Date time;
}

File: src/main/kotlin/org/shark/example/datasource/cassandra/pojo/MessageDo.kt
package org.shark.example.datasource.cassandra.pojo

import org.springframework.data.cassandra.core.mapping.Column
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table
import java.util.Date

@Table("message")
data class MessageDo(
    @PrimaryKey
    var messageKey: MessageKeyDo? = null,

    @Column("time")
    var time: Date? = null
)
